rules/CodeQuality/Rector/Class_/AddReturnTypeToDependedRector.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\PHPUnit\CodeQuality\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr;
+use PhpParser\Node\Stmt\Class_;
+use Rector\PhpParser\Node\BetterNodeFinder;
+use Rector\PHPStanStaticTypeMapper\Enum\TypeKind;
+use Rector\PHPUnit\NodeAnalyzer\TestsNodeAnalyzer;
+use Rector\Rector\AbstractRector;
+use Rector\StaticTypeMapper\StaticTypeMapper;
+use Rector\TypeDeclaration\TypeInferer\SilentVoidResolver;
+use Rector\TypePerfect\NodeFinder\ReturnNodeFinder;
```

Comment:
`Rector\TypePerfect` is not part of rector/rector

Reply:
instead, use ReturnsAnalyzer:

```php
        $returns = $this->betterNodeFinder->findReturnsScoped($functionLike);
        if (! $this->returnAnalyzer->hasOnlyReturnWithExpr($functionLike, $returns)) {
            return null;
        }
```

Reply:
Good point :+1: